ubereats/src/index.html

```diff
@@ -0,0 +1,310 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>UberEats</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" type="text/css" href="./styles/styles.css"/>
+  <link rel="stylesheet" type="text/css" href="./styles/normalize.css"/>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto">
+</head>
+<body>
+
+  <header class="header" id="hdr-top">
+    <div class="header-top">
+      <div class="container">
+        <div class="row">
+
+          <div class="col-xs-5 col-sm-2 col-md-2">
+            <a class="header-top__item header-logo" href="#">
+                <img class="header-logo__img" src="./images/logo.svg" alt="UberEats">
+            </a>
+          </div>
+
+          <form class="header-top__item header-location header-location--desktop">
+            <div class="header-location__item header-location__item--time">
+              <label class="header-location__label header-location__label--when">When</label>
+              <button class="header-location__field header-location__field--time">ASAP</button>
+            </div>
+            <div class="header-location__item header-location__item--destination">
+              <label class="header-location__label header-location__label--to">To</label>
+              <input class="header-location__field header-location__field--destination" type="text" placeholder="ul. Bolshaya Dmitrovka, 22" value="">
+            </div>
+          </form>
+
+          <div class="col-xs-7 col-sm-5 col-md-4">
+            <div class="header-button header-top__item">
+              <div class="header-button__acc">
+                <button class="header-button__link header-button__link--sign-up">Sign Up</button>
+                <button class="header-button__link header-button__link--register">Register</button>
+              </div>
+              <button class="header-button__basket"></button>
+            </div>
+          </div>
+
+        </div>
+      </div>
+    </div>
+
+    <div class="header-bottom">
+      <div class="container">
+        <div class="row">
+
+          <div class="col-xs-12">
```

Comment:
https://erodionov.ru/courses/verstka/FhTKluFzm7#%D0%BD%D0%B5%D1%82-%D0%BD%D1%83%D0%B6%D0%B4%D1%8B-%D0%B8%D1%81%D0%BF%D0%BE%D0%BB%D1%8C%D0%B7%D0%BE%D0%B2%D0%B0%D1%82%D1%8C-col-xs-12